python/pyiceberg/catalog/rest.py

```diff
@@ -175,11 +175,7 @@ class RestCatalog(Catalog):
     session: Session
     properties: Properties
 
-    def __init__(
-        self,
-        name: str,
-        **properties: str,
-    ):
+    def __init__(self, name: str, **properties: str):
```

Comment:
Why did this change? I always assumed that black was deterministic.

Reply:
I removed the trailing comma

Reply:
Nice!